.github/workflows/tag_image_push_rhel.yml

```diff
@@ -121,6 +132,27 @@ jobs:
         run: |
           docker login ${SCAN_REGISTRY} -u ${SCAN_REGISTRY_USER} -p ${SCAN_REGISTRY_PASSWORD}
 
+      - name: Check if LTS release
```

Comment:
nit - for clarity as only allies to EE
`Check if EE LTS release`
`Check if latest EE LTS release`

Reply:
We're checking in general if it is a LTS release, later on we decide that we tagged EE with `latest-lts` tag. For me it's a detail but changed: https://github.com/hazelcast/hazelcast-docker/pull/808/commits/67b76cac69881fa11487a9507282396eed1cc3c1